README.md

```diff
@@ -1,34 +1,148 @@
-# Hello World
-Hello World is a project about Introduction to Microverse
-# Built With
-it is built with Html & CSS
-# Key Features
-Html
-css
-# Getting Started
+<a name="readme-top"></a>
+
+
+
+<div align="center">
+
+  <img src="https://github.com/microverseinc/readme-template/blob/master/murple_logo.png" alt="logo" width="140" height="auto" />
+  <br/>
+
+  <h3><b>README.md</b></h3>
+
+</div>
+
+
+
+# 📗 Table of Contents
+
+- [📖 About the Project](#about-project)
+ - [🛠 Built With](#built-with)
+      - [💻 Getting Started](#getting-started)
+  - [Setup](#setup)
+  - [Prerequisites](#prerequisites)
+  - [Install](#install)
+  - [Usage](#usage)
+  - [Run tests](#run-tests)
+  - [👥 Authors](#authors)
+- [🤝 Contributing](#contributing)
+- [⭐️ Show your support](#support)
+- [🙏 Acknowledgements](#acknowledgements)
+- [📝 License](#license)
+
+
+# 📖 [Hello Microverse] <a name="about-project"></a>
+
+
+**[Hello Microverse]** Is a welcome to coding in Microverse
+## 🛠 Built With <a name="built-with"></a>
+    - HTML
+    - CSS
+
+<!-- LIVE DEMO -->
+
+## 🚀 Live Demo <a name="live-demo"></a>
+
+- #
```

Comment:
- [ ] Between the built with and live demo, there should be a `### Key Features <a name="key-features"></a>` section. Please add it back to your README file. Essentially, all sections of the Microverse README Template are compulsory except the FAQ section. 
And please include at least three features of what you have done. This is to make you used to best practices going forward. This project is a simple one and not many features were implemented, but you are expected to fill up this section with anything you like.